industrial_robot_client/src/joint_trajectory_action.cpp

```diff
@@ -123,6 +149,14 @@ void JointTrajectoryAction::goalCB(JointTractoryActionServer::GoalHandle gh)
     return;
   }
 
+  if (disabled_)
+  {
+    control_msgs::FollowJointTrajectoryResult rslt;
+    rslt.error_code = control_msgs::FollowJointTrajectoryResult::INVALID_GOAL;
+    gh.setRejected(rslt, "Robot has been disabled.  Goals will be ignored until the service is called to enable the robot.");
+    return;
+  }
```

Comment:
Could a `ROS_WARN(..)` be added here similar to how this is done below? I can already imagine the ROS Answers questions "my goals keep getting ignored, why isn't my robot moving !!11~~" from users not checking actionlib goal statuses.

Reply:
@pbeeson, could you address this request?

Reply:
Yep.  Done.